ambient-poly.py

```diff
@@ -120,12 +159,16 @@ def query(self):
             self.nodes[node].reportDrivers()
 
     def discover(self, *args, **kwargs):
+        '''
+        TODO: Maybe Make these discovered and not created by default
+        to support non PWS Ambient Weather consoles?
+        '''
         self.addNode(TemperatureNode(self, self.address, 'temperature', 'Temperatures'))
         self.addNode(HumidityNode(self, self.address, 'humidity', 'Humidity'))
         self.addNode(PressureNode(self, self.address, 'pressure', 'Barometric Pressure'))
         self.addNode(WindNode(self, self.address, 'wind', 'Wind'))
         self.addNode(PrecipitationNode(self, self.address, 'rain', 'Precipitation'))
-        self.addNode(LightNode(self, self.address, 'light', 'Illumination'))
+        self.addNode(LightNode(self, self.address, 'light', 'Illumination'))    
```

Comment:
Also looks like the only change was additional white space at the end of the line